clients/privacy-center/pages/api/fides-js.ts

```diff
@@ -392,8 +397,24 @@ export default async function handler(
   const { initialize: initializeQuery } = req.query;
   const skipInitialization = initializeQuery === "false";
 
+  // Check if fides.js is being loaded multiple times on the same page
+  const multipleLoadingMessage =
+    "FidesJS detected that it was already loaded on this page, aborting execution! Please use Fides.init() to reinitialize Fides, or disable this check to allow reloading with the 'fides_unsupported_repeated_script_loading' option";
+  const multiLoadCheck = `
+    if (
+      typeof window !== "undefined" &&
+      !!window.Fides &&
+      !!window.Fides.options &&
+      window.Fides.options.fidesUnsupportedRepeatedScriptLoading !==
+        "enabled_acknowledge_not_supported"
+    ) {
+      console.error("${multipleLoadingMessage}");
+      return;
+    }
+  `.replace(/(\r\n|\n|\r)/gm, ""); // remove newlines to avoid sourcemap issues!
```

Comment:
This needs to move into `fides.js` code itself; it's not _guaranteed_ that we always fetch FidesJS from the Privacy Center, we do bundle FidesJS independently in some cases.

Reply:
Good reminder! I'll make that change.